main.tf

```diff
@@ -80,6 +81,31 @@ data "aws_iam_policy_document" "default" {
       type        = "AWS"
       identifiers = ["*"]
     }
+  }]
+
+  statement = ["${flatten(data.aws_iam_policy_document.master.*.statement)}"]
+}
+
+data "aws_iam_policy_document" "master" {
```

Comment:
I can't see where you associate this policy with the bucket.

Also, heads up: https://github.com/hashicorp/terraform/issues/10543#issuecomment-265657860

Reply:
Here it is https://github.com/cloudposse/terraform-aws-s3-website/pull/10#discussion-diff-141544489R86
We do not attach policy, we get array of statements and include it to default policy document